dev/reference/tools/data-migration/manage-tasks.md

```diff
@@ -84,22 +92,17 @@ Flags:
 
 ## Manage the data replication task
 
-This section describes how to use the task management commands to execute the following operations:
-
-- [Create the data replication task](#create-the-data-replication-task)
-- [Check the data replication task status](#check-the-data-replication-task-status)
-- [Pause the data replication task](#pause-the-data-replication-task)
-- [Restart the data replication task](#restart-the-data-replication-task)
-- [Stop the data replication task](#stop-the-data-replication-task)
-- [Update the data replication task](#update-the-data-replication-task)
+This section describes how to use the task management commands to execute corresponding operations.
 
 ### Create the data replication task
 
-You can use the task management command to create the data replication task. Data Migration [prechecks the corresponding privileges and configuration automatically](/dev/reference/tools/data-migration/precheck.md) while starting the data replication.
+You can use the `start-task` command to create the data replication task. Data Migration [prechecks the corresponding privileges and configuration automatically](/dev/reference/tools/data-migration/precheck.md) while starting the data replication.
+
+{{< copyable "" >}}
 
 ```bash
-» help start-task
-start a task with config file
+help start-task
+start a task as defined in the config file
```

Comment:
we split the code block into two sections, one for input (`help start-task`) and another for output (`start a task ...`).

the same for other commands.

Reply:
@dcalvin Please address this comment. Thanks!